### DISCUSSION TITLE
【第四期飞桨黑客松】生态伙伴开源贡献——TVM项目

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50738] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#tvm)
大家可以在这里讨论有关TVM项目任务的问题哦～

**Comment by Leenorwich** [comment]
为了方便统计，选择基础题的参赛者进群啦
<img width="198" alt="企业微信截图_16777549108812" src="https://user-images.githubusercontent.com/122860301/222612047-4d268dca-d9b6-4bbc-966a-a8f9ddc362d4.png">

<div data-slate-node="element" data-sticky="true" class="mp-table-container" style="font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; print-color-adjust: exact; margin: 1px 0px; overflow: auto hidden; padding-bottom: 8px; padding-top: 32px; position: relative; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;PingFang SC&quot;, &quot;Helvetica Neue&quot;, &quot;Hiragino Sans GB&quot;, &quot;Microsoft YaHei&quot;, Arial, SimSun, sans-serif; font-size: 12px; padding-left: 0px;">

TVM | Team Name
-- | --
affine_channel |  
conv3d |  
data_norm |  
dist |  
eye |  
fill_zeros_like |  
gaussian_random |  
grid_sampler |  
index_select |  
mish |  
flip |  
p_norm |  
roi_align |  
share_data |  
silu |  
softmax_with_cross_entropy |  
softshrink |  
linspace |  
take_along_axis |  
thresholded_relu |  
tile |  
unique |  
unstack |  
where |  

</div>
